lib/bureaucrat/helpers.ex

```diff
@@ -125,24 +125,58 @@ defmodule Bureaucrat.Helpers do
       opts =
         opts
         |> Keyword.put_new(:operation_id, default_operation_id)
+        |> Keyword.update!(:description, &(&1 || default_description()))
+
+      if Keyword.fetch!(opts, :description) != nil do
+        Bureaucrat.Recorder.doc(conn, opts)
+      else
+        file = Keyword.fetch!(opts, :file)
+        line = Keyword.fetch!(opts, :line)
+
+        Mix.shell().info("""
+        The request at #{file}:#{line} won't be recorded by bureaucrat because
+        the description can't be determined and none is explicitly provided.
+        To address this, you can pass the :description option to this macro.
+
+        If this macro is invoked indirectly, via the request macros, such as
+        get and post, you can switch to the _undocumented version to
+        explicitly avoid generating the documentation.
+
+        Alternatively, you can provide the description manually with something
+        like doc(conn, description: "some description"), where conn is the result
+        of the _undocumented macro.
```

Comment:
Should we indicate that the default case is to set the description based off the test name?

Reply:
Why not use `Logger`?

Reply:
I was on the fence here, and I'm still not completely sure what's more appropriate. My reasoning was that logging is something you want to do in a running system (dev/staging/prod), while this is being done only in tests. But I don't really have a strong opinion.